audio/src/main/java/dev/masonroot/audio/exceptions/PiperInitializationException.java

```diff
@@ -0,0 +1,39 @@
+package dev.masonroot.audio.exceptions;
+
+import java.nio.file.Path;
+
+/**
+ * Exception thrown when the Piper TTS engine fails to initialize with the specified detail message.
+ *
+ * <p>This exception is used to indicate issues during the initialization of the Piper
+ * text-to-speech engine, such as when the model path is invalid or the model cannot be loaded.
+ *
+ * <p><b>Notes:</b>
+ *
+ * <ul>
+ *   <li>This exception extends {@code EngineInitializationException} to provide more specific error
+ *       information related to Piper initialization.
+ * </ul>
+ */
+public final class PiperInitializationException extends EngineInitializationException {
```

Comment:
This exception should have a constructor which accepts the cause so that the stack trace contains the most amount of relevant information for debugging. Currently information is lost when this exception is being thrown as seen in `PiperTtsEngine` on line 60 where the causing `IOException` is erased.